Sources/SwiftDocC/Benchmark/Metrics/PeakMemory.swift

```diff
@@ -54,22 +55,25 @@ extension Benchmark {
                     .components(separatedBy: CharacterSet.decimalDigits.inverted)
                     .filter({ !$0.isEmpty })
                     .first,
-                let peakMemory = Double(peakMemoryString) else { return nil }
+                let peakMemory = Double(peakMemoryString)
+            else { return nil }
 
-            return Int64(peakMemory * 1024) // convert from KBytes to bytes
+            return Int64(peakMemory * 1024)  // convert from KBytes to bytes
```

Comment:
Personal preference: I don't really like the "two spaces before end-of-line comments" behavior here. It looks like that's configurable, though:

```
"spacesBeforeEndOfLineComments": 1
```